sql/src/main/java/io/crate/action/sql/TransportSQLAction.java

```diff
@@ -155,10 +155,11 @@ private SQLResponse createResponse(Analysis analysis, SQLRequest request, List<O
             }
         } else {
             assert analysis.rootRelation() != null;
-            outputNames = new String[analysis.rootRelation().fields().size()];
-            outputTypes = new DataType[analysis.rootRelation().fields().size()];
-            for (int i = 0; i < analysis.rootRelation().fields().size(); i++) {
-                Field field = analysis.rootRelation().fields().get(i);
+            List<Field> fields = analysis.rootRelation().fields();
+            outputNames = new String[fields.size()];
```

Comment:
In the futre please avoid style changes like this or do these kind of changes in smaller PRs against master directly..  Changes like this often result in unnecessary merge conflicts otherwise.